modules/claims_api/app/models/claims_api/auto_established_claim.rb

```diff
@@ -49,7 +49,7 @@ def to_internal
 
     def self.pending?(id)
```

Comment:
Since we modified this method, did we capture all of the instances where this gets triggered properly?

Reply:
yeah, the base controller is the only spot it presently is called 👍